tests/unit/test_avrotools.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests the avrotools module
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+import ga4gh.avrotools as avrotools
+import ga4gh.protocol as protocol
+import tests.utils as utils
+
+
+class TestValidationTool(unittest.TestCase):
+    """
+    Tests the ValidationTool
+    """
+    def testNonProtocolElement(self):
+        """
```

Comment:
The comments are great, but can we change these to non-docstrings please (i.e., just ordinary comments)? Unittest uses the docstrings when you run nosetest -v, and it can make it hard to figure out what test is doing what.